viewer-prototype/src/node/trace-server-service.ts

```diff
@@ -4,8 +4,13 @@ import { TraceServerConfigService } from '../common/trace-server-config';
 @injectable()
 export class TraceServerServiceImpl implements TraceServerConfigService {
 
-    async startTraceServer(path: string | undefined, port: number | undefined): Promise<void> {
-        spawn(`${path}`, ['-vmargs', `-Dtraceserver.port=${port}`]);
+    startTraceServer(path: string | undefined, port: number | undefined): Promise<true | string> {
```

Comment:
So the Promise returns `true` on success and an error message on failure? Can you please document that?
Alternatively, wouldn't it be simpler to always just return an error string (empty on success)?

Reply:
Modified the logic.